### PR TITLE
Added docstring for sets property method

### PR DESCRIPTION
**Pull request by jremes-foss**
Hello.

```sets``` property method in ProductSet class of sympy/sets/sets.py was missing a docstring. This has been now added, with an appropriate example.

```./bin/test quality``` executed locally - all green.

Thank you.

### CODE REVIEW

File: sympy/sets/sets.py [hunk 1]
```diff
@@ -706,6 +706,19 @@ def _union(self, other):
...
     @property
     def sets(self):
+        """
+        Returns the arguments of 'self', or in the other words, the factors
+        of Cartesian Product.
+
+        Examples
+        ========
+
+        >>> from sympy import Interval, ProductSet
```
Comment (gxyd) [inline sympy/sets/sets.py]: `ProductSet` isn't used nowhere.

Comment (Newman101) [reply sympy/sets/sets.py]: Thanks for the note, fixing this now.

File: sympy/sets/sets.py [hunk 2]
```diff
@@ -706,6 +706,19 @@ def _union(self, other):
 
     @property
     def sets(self):
+        """
+        Returns the arguments of 'self', or in the other words, the factors
```
Comment (gxyd) [inline sympy/sets/sets.py]: 'the' factors? This should better be `a tuple of factors`. Are the factors of cartesian product in maths unique?

**Review by gxyd — COMMENTED** [review]
The added docstring rather sounds wrong to me. I am -1 for this. Feel free to improve upon this and re-open this.

**Pull request closed without merging**